### PR TITLE
Saptune summary frontend

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Add saptune summary box to HostDetails view. I have put some of the elements inside `SaptuneDetails`, as the will be used there, once we have that component. It looks the more appropriate place.

The used "struct" of `saptuneStatus` is temporal. It will be similar, we might need to change it once we exactly know how the redux store will be composed.

Storybook (in real execution the fields are like `Not installed`):
image

## How was this tested?

Tested and stories added.

### CODE REVIEW

File: assets/js/components/HostDetails/HostDetails.jsx [hunk 1]
```diff
@@ -140,18 +151,63 @@ function HostDetails({
...
+                  content: <SaptuneTuningState state={saptuneTuning} />,
+                },
+                {
+                  title: 'Configured Version',
+                  content: saptuneConfigureVersion || '-',
+                },
+              ]}
+            />
+          </div>
+          <div className="mt-4 bg-white shadow rounded-lg py-4 xl:w-1/4">
```
Comment (arbulu89) [inline assets/js/components/HostDetails/HostDetails.jsx:200]: @dottorblaster @nelsonkopliku I guess we will need to have some `checksAvailable` kind of variable to decide if we put this `coming soon` or the results box.

File: assets/js/components/HostDetails/HostDetailsPage.jsx [hunk 2]
```diff
@@ -63,9 +63,11 @@ function HostDetailsPage() {
       heartbeat={host.heartbeat}
       hostID={host.id}
       hostname={host.hostname}
+      ipAddresses={host.ip_addresses}
       provider={host.provider}
       providerData={host.provider_data}
       sapInstances={sapInstances}
+      saptuneStatus={{}}
```
Comment (arbulu89) [inline assets/js/components/HostDetails/HostDetailsPage.jsx:70]: Temporary empty

File: assets/js/components/HostDetails/HostDetails.jsx (hunk 1 above)
Comment (nelsonkopliku) [reply assets/js/components/HostDetails/HostDetails.jsx:200]: I guess, so. We'll be working on that soon.

**Comment by abravosuse** [comment]
@arbulu89 I see that you have assigned this to me. The storybooks look good. Is there anything else that you would like me to do here?

File: assets/js/lib/test-utils/factories/hosts.js [hunk 3]
```diff
@@ -58,5 +69,6 @@ export const hostFactory = Factory.define(({ params, sequence }) => {
     sles_subscriptions: slesSubscriptionFactory.buildList(4, { host_id: id }),
     deregisterable: false,
     selected_checks: [],
+    saptuneStatus: saptuneStatusFactory.build(),
```
Comment (arbulu89) [inline assets/js/lib/test-utils/factories/hosts.js:72]: Need to change this by `saptune_status`

**Review by CDimonaco — COMMENTED** [review]
just some general feedback

**Review by CDimonaco — APPROVED** [review]
LGTM

**Pull request merged**